lib/apiaria.rb

```diff
@@ -28,4 +28,27 @@ def get_mentor_availability(mentor_id)
     JSON.parse(response.body)
   end
   
+  def get_messages(pageNum = nil)
+    unless pageNum
+      response = self.class.get("/message_threads",  headers: { "authorization" => @auth_token })
+      JSON.parse(response.body)
+    else
+      url = "/message_threads?page=" + pageNum.to_s
+      response = self.class.get(url,  headers: { "authorization" => @auth_token })
+      JSON.parse(response.body)
+    end
+  end
+
+  def create_message(sender, recipientId, body, options = {})
```

Comment:
@wonderwoman13 I just used the solution you recommended,though still not convinced why my wouldn't work though :(  .I guess I'll go with the flow for now.

Even with the simpler solution,I see the below error in irb, but the messages are showing up consistently -

`2.3.0 :003 > p.create_message("praveenyr@gmail.com",623967,"Lets hit it" ,"Beach!" )
JSON::ParserError: A JSON text must at least contain two octets!
        from /usr/local/rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/json/common.rb:156:in `initialize'
        from /usr/local/rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/json/common.rb:156:in `new'
        from /usr/local/rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/json/common.rb:156:in `parse'
        from /home/ubuntu/workspace/lib/apiaria.rb:59:in `create_message'
        from (irb):3
        from /usr/local/rvm/rubies/ruby-2.3.0/bin/irb:11:in `<main>'`